modules/installation-configuration-parameters.adoc

```diff
@@ -529,11 +530,14 @@ ifndef::aws,bare,gcp,ibm-z,ibm-power,azure,ibm-power-vs[]
 |String
 endif::aws,bare,gcp,ibm-z,ibm-power,azure,ibm-power-vs[]
 
-ifdef::aws,bare,azure,gcp[]
+ifdef::aws,azure,gcp,bare[]
 |`controlPlane.architecture`
-|Determines the instruction set architecture of the machines in the pool. Currently, clusters with varied architectures are not supported. All pools must specify the same architecture. Valid values are `amd64` and `arm64`. See _Supported installation methods for different platforms_ in _Installing_ documentation for information about instance availability.
+|Determines the instruction set architecture of the machines in the pool. Currently, clusters with varied architectures are not supported. All pools must specify the same architecture. Valid values are `amd64` and `arm64`. 
 |String
-endif::aws,bare,azure,gcp[]
+ifdef::aws,azure[]
+ Not all installation options support the 64-bit ARM architecture. To verify if your installation option is supported on your platform, see _Supported installation methods for different platforms_ in _Selecting a cluster installation method and preparing it for users_.
```

Comment:
I believe the condition is in the wrong spot. It needs to be included in the column defined by line 535.